spec/requests/get_food_spec.rb

```diff
@@ -62,7 +62,7 @@
       post get_food_path
 
       expect(response.body).to have_content(I18n.t("coronavirus_form.groups.getting_food.questions.get_food.title"))
-      expect(response.body).to have_content(I18n.t("coronavirus_form.errors.radio_field", field: I18n.t("coronavirus_form.groups.getting_food.questions.get_food.title").downcase))
+      expect(response.body).to have_content(I18n.t("coronavirus_form.groups.getting_food.questions.get_food.custom_select_error", field: I18n.t("coronavirus_form.groups.getting_food.questions.get_food.title").downcase))
```

Comment:
You can also remove the `field` now we're using a custom error
```suggestion
      expect(response.body).to have_content(I18n.t("coronavirus_form.groups.getting_food.questions.get_food.custom_select_error"))
```

Reply:
Good spot - thanks! I've made the changes and pushed again.